ironfish/src/rpc/routes/chain/getTransactionStream.ts

```diff
@@ -162,7 +162,7 @@ router.register<typeof GetTransactionStreamRequestSchema, GetTransactionStreamRe
               value: CurrencyUtils.encode(decryptedNote.value()),
               memo: decryptedNote.memo(),
               assetId: decryptedNote.assetId().toString('hex'),
-              assetName: assetValue?.name.toString('hex') || '',
+              assetName: assetValue ? BufferUtils.toHuman(assetValue.name) : '',
```

Comment:
This behavior is slightly inaccurate since the asset name is stored as a buffer in storage, on transaction descriptions, and on the asset itself. Consumers of this RPC wouldn't be able to get the correct value with this change. Is there another way we can address this bug?